include/deal.II/base/vectorization.h

```diff
@@ -3841,9 +3841,25 @@ inline DEAL_II_ALWAYS_INLINE VectorizedArray<Number, width>
   return VectorizedArray<Number, width>() - u;
 }
 
+/**
+ * Output operator for vectorized array.
+ *
+ * @relatesalso VectorizedArray
+ */
+template <typename Number>
+inline std::ostream &
+operator<<(std::ostream &out, const dealii::VectorizedArray<Number> &p)
+{
+  for (unsigned int i = 0;
+       i < dealii::VectorizedArray<Number>::n_array_elements;
+       ++i)
+    out << p[i] << ' ';
+  out << '\n';
```

Comment:
The question is whether we want to skip the newline for consistency with the output in the `Point` and `Tensor` classes, see e.g.
https://github.com/dealii/dealii/blob/master/include/deal.II/base/point.h#L669-L673
While I would agree that 16 floats definitely fill a line, I think it is better to let the user decide how to wrap things.

Reply:
Sure, that's fine with me.